oras/provider.py

```diff
@@ -250,8 +253,13 @@ def upload_blob(
                 blob, container, layer, refresh_headers=refresh_headers
```

Comment:
Please see the comment above:

```
 # Chunked for large, otherwise POST and PUT
# This is currently disabled unless the user asks for it, as
# it doesn't seem to work for all registries
```
Chunked was not working well for different registries so I am hesitant to add it back.